typed-racket-lib/typed-racket/base-env/prims.rkt

```diff
@@ -1322,7 +807,12 @@ This file defines two sorts of primitives. All of them are provided into any mod
                                     (set! vs new-vs)]
                  [else  (unsafe-vector-set! vs i v)])
            (set! i (unsafe-fx+ i 1)))
-         (vector-copy vs 0 i)))]))
```

Comment:
May want to add a comment saying that this is intentionally inlining `vector-copy` to avoid the dependency, to avoid reintroducing it on code cleanup.